src/cluster_info.rs

```diff
@@ -65,6 +73,20 @@ pub struct ClusterInfo {
     keypair: Arc<Keypair>,
 }
 
+#[derive(Default, Clone)]
+pub struct Locality {
+    /// The bounds of the neighborhood represented by this locality
+    pub neighbor_bounds: (usize, usize),
```

Comment:
Do these fields need to be `pub`?

Reply:
Yeah, they're being used by callers of localize() outside of this module.